CMakeLists.txt

```diff
@@ -63,6 +63,7 @@ option(WITH_DLT_SHM_ENABLE    "EXPERIMENTAL! Set to ON to use shared memory as I
 option(WITH_DLT_ADAPTOR       "Set to ON to build src/adaptor binaries"                                          OFF)
 option(WITH_DLT_CONSOLE       "Set to ON to build src/console binaries"                                          ON)
 option(WITH_DLT_EXAMPLES      "Set to ON to build src/examples binaries"                                         ON)
+option(WITH_DLT_FILETRANSFER  "Set to ON to build dlt-system with filetransfer support"                          ON)
```

Comment:
I would like to set the default to OFF because WITH_DLT_SYSTEM is set to OFF and this only works on Linux.

Reply:
ACK

Reply:
@oncaphillis Looks like you've changed the wrong line to OFF...?

Reply:
:-(

Reply:
ACK again